diff --git a/base/BUILD.gn b/base/BUILD.gn
index a9fb242620bf572ed1e3bba745165e39f00eda73..e78999d4f8636732bf21cbb49cfbb9d98fdcba04 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -3895,6 +3895,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/TaskTraits.java",
       "android/java/src/org/chromium/base/task/TaskTraitsExtensionDescriptor.java",
     ]
+    sources += brave_java_base_sources
 
     annotation_processor_deps = [ "//base/android/jni_generator:jni_processor" ]
 
