diff --git a/build/config/android/config.gni b/build/config/android/config.gni
index c4ce2db904c438bd9d5a38dd52a6e61421c2d56e..2b157dc38f80bf3d8110daae59e921aec65a7ccd 100644
--- a/build/config/android/config.gni
+++ b/build/config/android/config.gni
@@ -3,6 +3,7 @@
 # found in the LICENSE file.
 
 # This file contains common system config stuff for the Android build.
+import("//brave/build/android/config.gni")
 
 # NOTE: Because Chrome OS builds may depend on targets built with the Android
 # toolchain, this GNI file may be read and processed from within Chrome OS
