diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index 0eb315b1cc862aecdfab068af2f1729a97deca2a..aa9f1fc4737fe63047b548b7458dcf285997a7f6 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1703,6 +1703,7 @@ config("chromium_code") {
       cflags += [ "-Wextra-semi" ]
     }
   }
+  cflags -= brave_removed_chromium_code_cflags
 
   configs = [ ":default_warnings" ]
 }
