diff --git a/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java b/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java
index b585362814b820a99e7a8046d63fd0cc9ab0207e..2bfe1599325ac3ff6d6074ecc9a218b16e77cba1 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java
@@ -367,7 +367,7 @@ public abstract class ChromeActivity<C extends ChromeActivityComponent>
             new ArrayList<>();

     protected ChromeActivity() {
-        mIntentHandler = new IntentHandler(this, createIntentHandlerDelegate());
+        mIntentHandler = new org.chromium.chrome.browser.BraveIntentHandler(this, createIntentHandlerDelegate());
     }

     @Override
