diff --git a/chrome/browser/resources/settings/people_page/sync_controls.js b/chrome/browser/resources/settings/people_page/sync_controls.js
index 13062cf9d434ccba33c7fe70da112a66ce085ee0..41d54f1731edd58d1a0f5ef3c81ad46cc482b63a 100644
--- a/chrome/browser/resources/settings/people_page/sync_controls.js
+++ b/chrome/browser/resources/settings/people_page/sync_controls.js
@@ -91,7 +91,7 @@ Polymer({
     const router = settings.Router.getInstance();
     if (router.getCurrentRoute() ===
         /** @type {{ SYNC_ADVANCED: !settings.Route }} */
-        (router.getRoutes()).SYNC_ADVANCED) {
+        (router.getRoutes()).BRAVE_SYNC_SETUP) {
       this.browserProxy_.didNavigateToSyncPage();
     }
   },
@@ -224,6 +224,7 @@ Polymer({
    * @private
    */
   syncControlsHidden_() {
+    return false;
     if (!this.syncStatus) {
       // Show sync controls by default.
       return false;
