diff --git a/chrome/renderer/BUILD.gn b/chrome/renderer/BUILD.gn
index 7f12b3bb1eef8d47fed4846e947d788faa8b843d..1e2e0ab37d4d086f800139a553476906b0a17756 100644
--- a/chrome/renderer/BUILD.gn
+++ b/chrome/renderer/BUILD.gn
@@ -225,6 +225,7 @@ static_library("renderer") {
     "//v8",
   ]
 
+  sources += brave_chrome_renderer_sources public_deps += brave_chrome_renderer_public_deps
   data_deps = [ "//tools/v8_context_snapshot" ]
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
