diff --git a/chrome/browser/notifications/notification_platform_bridge_mac.mm b/chrome/browser/notifications/notification_platform_bridge_mac.mm
index 437477b8f697234f06da0909d20d6d957df5fd38..472ad8809f762e1090a1996a3e061fe157d145e4 100644
--- a/chrome/browser/notifications/notification_platform_bridge_mac.mm
+++ b/chrome/browser/notifications/notification_platform_bridge_mac.mm
@@ -188,6 +188,7 @@ void NotificationPlatformBridgeMac::Display(
   } else {
     NSUserNotification* toast = [builder buildUserNotification];
     [notification_center_ deliverNotification:toast];
+    BRAVE_DISPLAY_
   }
 }
 
