diff --git a/chrome/browser/resources/settings/BUILD.gn b/chrome/browser/resources/settings/BUILD.gn
index 6a612c2bb3c09dc0e6c122ece80d74a11c988ca3..3ddf9e174fecc9eed982003cf842fa9ead7aa99a 100644
--- a/chrome/browser/resources/settings/BUILD.gn
+++ b/chrome/browser/resources/settings/BUILD.gn
@@ -46,6 +46,7 @@ if (optimize_webui) {
       "chrome://resources/mojo/mojo/public/js/bindings.js",
       "chrome://resources/mojo/skia/public/mojom/skcolor.mojom-webui.js",
     ]
+    deps += brave_deps_chrome_browser_resources_settings_build
   }
 }
 
@@ -93,6 +94,7 @@ generate_grd("build_grd") {
       "$target_gen_dir/$preprocess_gen_manifest",
     ]
   }
+  deps += [ "//brave/browser/resources/settings:resources" ]
 }
 
 preprocess_if_expr("preprocess") {
