diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index e16f09fb8497206b02f76b0adcdc816fc4808a25..d87ae6f7c4b73e70dbb042caa09572657dd84a08 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -903,6 +903,7 @@ if (!is_android) {
       "//v8",
     ]
 
+    deps += [ "//brave/test:brave_browser_tests_deps", ]
     # Runtime dependencies
     data_deps = [
       "//chrome:browser_tests_pak",
