diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 544a4af1fe37e294804949e4e8b3bf04c7f33ca2..2a27473b705347acbd35cd1ed065ca659d563de0 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -898,6 +898,7 @@ if (!is_android) {
       "//v8",
     ]

+    deps += [ "//brave/test:brave_browser_tests_deps", ]
     # Runtime dependencies
     data_deps = [
       "//chrome:browser_tests_pak",
