diff --git a/chrome/browser/notifications/notification_platform_bridge_mac_utils.mm b/chrome/browser/notifications/notification_platform_bridge_mac_utils.mm
index 6bb2731db1dd19b636f681dc2fb9568ab9acef2e..0499fcfd56ae165c248090101a0b462d3d543975 100644
--- a/chrome/browser/notifications/notification_platform_bridge_mac_utils.mm
+++ b/chrome/browser/notifications/notification_platform_bridge_mac_utils.mm
@@ -59,6 +59,7 @@ void DoProcessMacNotificationResponse(
 // call this method directly as SysInfo::OperatingSystemVersionNumbers might be
 // an expensive call. Instead use SupportsAlerts which caches this value.
 bool MacOSSupportsXPCAlertsImpl() {
+  BRAVE_SUPPORTSALERTSIMPL_
   int32_t major, minor, bugfix;
   base::SysInfo::OperatingSystemVersionNumbers(&major, &minor, &bugfix);
   // Allow alerts on all versions except 10.15.0, 10.15.1 & 10.15.2.
