diff --git a/chrome/browser/flags/android/chrome_feature_list.cc b/chrome/browser/flags/android/chrome_feature_list.cc
index a08e9bfc3405baf24826b3bdb233c6cbd9c9e300..26766281d8b407166ada39bb4d62bc1d6518fdd7 100644
--- a/chrome/browser/flags/android/chrome_feature_list.cc
+++ b/chrome/browser/flags/android/chrome_feature_list.cc
@@ -314,6 +314,7 @@ const base::Feature* kFeaturesExposedToJava[] = {
 };

 const base::Feature* FindFeatureExposedToJava(const std::string& feature_name) {
+  BRAVE_FIND_FEATURE_EXPOSED_TO_JAVA
   for (const auto* feature : kFeaturesExposedToJava) {
     if (feature->name == feature_name)
       return feature;
