diff --git a/chrome/browser/ui/views/translate/translate_bubble_view.h b/chrome/browser/ui/views/translate/translate_bubble_view.h
index 7d37d69b9591c1ec5635882a1434f928ee0ae399..81145c084c553f13eb06229f5009a06e2d51f445 100644
--- a/chrome/browser/ui/views/translate/translate_bubble_view.h
+++ b/chrome/browser/ui/views/translate/translate_bubble_view.h
@@ -95,6 +95,7 @@ class TranslateBubbleView : public LocationBarBubbleDelegateView,
 
   // Returns the current view state.
   TranslateBubbleModel::ViewState GetViewState() const;
+  BRAVE_TRANSLATE_BUBBLE_VIEW_H_
 
  protected:
   // LocationBarBubbleDelegateView:
