diff --git a/chrome/browser/ui/browser.h b/chrome/browser/ui/browser.h
index dc607e49cc05d4c0ba5458de629ba635a2166f07..e489593ffd5146981cdf1701fd23715731f4e478 100644
--- a/chrome/browser/ui/browser.h
+++ b/chrome/browser/ui/browser.h
@@ -702,6 +702,7 @@ class Browser : public TabStripModelObserver,
 
   StatusBubble* GetStatusBubbleForTesting();
 
+  BRAVE_BROWSER_H
  private:
   friend class BrowserTest;
   friend class ExclusiveAccessTest;
