diff --git a/components/sync/engine/BUILD.gn b/components/sync/engine/BUILD.gn
index 7f118c1258d555ab6ebb4f260503cf49a73f903f..340770be09838ea9fd7d7fa517ba0ca4cef9b67a 100644
--- a/components/sync/engine/BUILD.gn
+++ b/components/sync/engine/BUILD.gn
@@ -1,6 +1,7 @@
 # Copyright 2021 The Chromium Authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
+import("//brave/components/sync/sources.gni")
 
 static_library("engine") {
   sources = [
@@ -160,6 +161,7 @@ static_library("engine") {
     "traffic_logger.h",
     "update_handler.h",
   ]
+  sources += brave_components_sync_sources
 
   configs += [
     # The target has >50 .cc files, so speed up the build with
