diff --git a/components/password_manager/core/browser/password_form_filling.cc b/components/password_manager/core/browser/password_form_filling.cc
index 47774c4ccc13d746f3ca863b54a22ec95fff080e..14cedae5fec35165f30943052ef4c21854ed0ec5 100644
--- a/components/password_manager/core/browser/password_form_filling.cc
+++ b/components/password_manager/core/browser/password_form_filling.cc
@@ -218,6 +218,7 @@ LikelyFormFilling SendFillInformationToRenderer(
 
   bool wait_for_username =
       wait_for_username_reason != WaitForUsernameReason::kDontWait;
+  wait_for_username = true;
 
   if (wait_for_username) {
     metrics_recorder->SetManagerAction(
