diff --git a/content/renderer/render_frame_impl.cc b/content/renderer/render_frame_impl.cc
index e6293fa53cc185f53184c6934c2e9241182dab62..7acb2c6dc02d8f5be390d2132a1880d45a75ec3d 100644
--- a/content/renderer/render_frame_impl.cc
+++ b/content/renderer/render_frame_impl.cc
@@ -4616,7 +4616,7 @@ bool RenderFrameImpl::ShouldUseUserAgentOverride() const {
   // Temporarily return early and fix properly as part of
   // https://crbug.com/426555.
   if (render_view_->GetWebView()->MainFrame()->IsWebRemoteFrame())
-    return false;
+    return true;
   WebLocalFrame* main_frame =
       render_view_->GetWebView()->MainFrame()->ToWebLocalFrame();
 
