diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index b094f872a16f019f0eba61164880cbaba57be562..46530df24c6bb7f9ec94659bbd9d7e60ce7537b1 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -42,6 +42,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
