diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index 07c6941bb6cdb81be4c4e46eb7c5d7fcfcfe7b80..d37a6d92e44255a5108b93d6126903399982bb27 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -42,6 +42,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
