diff --git a/third_party/blink/renderer/modules/webaudio/realtime_analyser.h b/third_party/blink/renderer/modules/webaudio/realtime_analyser.h
index 47c325d0707904ec7d62ffc5dc4f32f03605af59..241e19952b5e7199f126ad4313ea941a784de81e 100644
--- a/third_party/blink/renderer/modules/webaudio/realtime_analyser.h
+++ b/third_party/blink/renderer/modules/webaudio/realtime_analyser.h
@@ -73,6 +73,7 @@ class RealtimeAnalyser final {
   static const unsigned kMaxFFTSize;
   static const unsigned kInputBufferSize;
 
+  BRAVE_REALTIMEANALYSER_H
  private:
   // The audio thread writes the input audio here.
   AudioFloatArray input_buffer_;
