diff --git a/third_party/blink/renderer/core/BUILD.gn b/third_party/blink/renderer/core/BUILD.gn
index dfb30a8e9396469798a499b5b74bab5f7b3ce88c..8662819e3e300122a872f3529927be58bbeaccf2 100644
--- a/third_party/blink/renderer/core/BUILD.gn
+++ b/third_party/blink/renderer/core/BUILD.gn
@@ -173,6 +173,7 @@ component("core") {
 
   visibility = []  # Allow re-assignment of list.
   visibility = [ "//third_party/blink/*" ]
+  visibility += brave_blink_renderer_core_visibility
 
   sources = [
     "core_initializer.cc",
@@ -274,6 +275,7 @@ component("core") {
     "//v8",
   ]
 
+  public_deps += brave_blink_renderer_core_public_deps
   deps = [
     "//build:chromeos_buildflags",
     "//components/paint_preview/common",
