diff --git a/third_party/blink/common/client_hints/client_hints.cc b/third_party/blink/common/client_hints/client_hints.cc
index 833602b7c73a8edc037410eaf578af870a968df2..8dbfe82b96b8e0863d55d1976f18d9d8e07619cf 100644
--- a/third_party/blink/common/client_hints/client_hints.cc
+++ b/third_party/blink/common/client_hints/client_hints.cc
@@ -127,6 +127,7 @@ base::Optional<std::vector<network::mojom::WebClientHintsType>> FilterAcceptCH(
 }
 
 bool IsClientHintSentByDefault(network::mojom::WebClientHintsType type) {
+  BRAVE_CLIENT_HINTS_IS_CLIENT_HINT_SENT_BY_DEFAULT
   return (type == network::mojom::WebClientHintsType::kUA ||
           type == network::mojom::WebClientHintsType::kUAMobile);
 }
