diff --git a/tools/gritsettings/resource_ids.spec b/tools/gritsettings/resource_ids.spec
index be4bc5cf81ceb15b5f5695e1f3cb3c78fa56b7f4..61b6e1d8755634765531ef3208f0c04ea3804260 100644
--- a/tools/gritsettings/resource_ids.spec
+++ b/tools/gritsettings/resource_ids.spec
@@ -47,6 +47,9 @@
   "chrome/app/chromium_strings.grd": {
     "messages": [400],
   },
+  "chrome/app/brave_strings.grd": {
+    "messages": [400],
+  },
   "chrome/app/google_chrome_strings.grd": {
     "messages": [400],
   },
@@ -454,6 +457,9 @@
     "META": {"align": 100},
     "messages": [2700],
   },
+  "components/components_brave_strings.grd": {
+    "messages": [2700],
+  },
   "components/components_google_chrome_strings.grd": {
     "messages": [2700],
   },
