diff --git a/ui/native_theme/native_theme.h b/ui/native_theme/native_theme.h
index 108b6d6d4d23166d8cc1b674127596b49e8436a0..6f5f8895339583f514b8633a6ca04d8e1e65896c 100644
--- a/ui/native_theme/native_theme.h
+++ b/ui/native_theme/native_theme.h
@@ -455,6 +455,7 @@ class NATIVE_THEME_EXPORT NativeTheme {
 
   // Returns the system's caption style.
   virtual base::Optional<CaptionStyle> GetSystemCaptionStyle() const;
+  BRAVE_UI_NATIVE_THEME_NATIVE_THEME_H_
 
   virtual ColorScheme GetDefaultSystemColorScheme() const;
 
