diff --git a/ui/native_theme/native_theme_mac.mm b/ui/native_theme/native_theme_mac.mm
index 43ba9880a4861587c9a10d054d32a8e39a457054..ed57dee16b86a669d72d48f7d8a2610d0ad78aec 100644
--- a/ui/native_theme/native_theme_mac.mm
+++ b/ui/native_theme/native_theme_mac.mm
@@ -191,6 +191,7 @@ base::Optional<SkColor> NativeThemeMac::GetOSColor(
 
   // Even with --secondary-ui-md, menus use the platform colors and styling, and
   // Mac has a couple of specific color overrides, documented below.
+  GET_BRAVE_COLOR(color_id)
   switch (color_id) {
     case kColorId_EnabledMenuItemForegroundColor:
       return skia::NSSystemColorToSkColor([NSColor controlTextColor]);
